src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -497,52 +497,119 @@ impl AstDisplay for DbzMode {
 }
 impl_display!(DbzMode);
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum KafkaSecurityOptions {
+    SSL {
+        key: Option<UnresolvedObjectName>,
```

Comment:
Should these be object names, string literals, or pseudo-string literals via `Ident`?

Reply:
Makes sense! I hadn't realized that object references were, in fact, part of this feature extension